jscomp/syntax/tests/printer/expr/expected/dict.res.txt

```diff
@@ -17,14 +17,10 @@ let x = dict{
   "baz": baz,
 }
 
-let x = dict{"foo": "bar", "bar": "baz"}
-let x = dict{"foo": "bar", "bar": "baz", "baz": baz}
+let x = Js.Dict.fromArray([("foo", "bar"), ("bar", "baz")])
```

Comment:
Can you remove
```
let x = Js.Dict.fromArray([("foo", "bar"), ("bar", "baz")])
let x = Js.Dict.fromArray([("foo", "bar"), ("bar", "baz"), ("baz", baz)])

let x = Js.Dict.fromArray([
        ("foo", "bar"),
        ("bar", "baz"),
        ("baz", baz)
])
```
from the test source? This only creates confusion.

Reply:
Yes, removing.